prisma-fmt/src/text_document_completion.rs

```diff
@@ -66,12 +97,39 @@ fn push_ast_completions(
                 completion_list.items.push(CompletionItem {
                     label: referential_action.as_str().to_owned(),
                     kind: Some(CompletionItemKind::ENUM),
-                    // what is the difference between detail and documentation?
                     detail: Some(referential_action.documentation().to_owned()),
                     ..Default::default()
                 });
             }
         }
+
+        ast::SchemaPosition::DataSource(_source_id, ast::SourcePosition::Source) => {
+            if connector.has_capability(psl::datamodel_connector::ConnectorCapability::MultiSchema)
+                && schemas_span.is_none()
+                && preview_features.contains(PreviewFeature::MultiSchema)
+            {
+                completion_list.items.push(CompletionItem {
+                    label: "schemas".to_owned(),
+                    insert_text: Some(r#"schemas = [$0]"#.to_owned()),
+                    insert_text_format: Some(InsertTextFormat::SNIPPET),
+                    kind: Some(CompletionItemKind::PROPERTY),
```

Comment:
```suggestion
                    kind: Some(CompletionItemKind::FIELD),
```

Currently, all the `dataSourceFields` are "CompletionItemKind.Field"
```
export const supportedDataSourceFields: CompletionItem[] = convertToCompletionItems(
  completions.dataSourceFields,
  CompletionItemKind.Field,
)
```

We can change that if we want but not mix them, currently it looks like 
<img width="700" alt="Screenshot 2023-01-10 at 11 00 43" src="https://user-images.githubusercontent.com/1328733/211521754-565f6acd-f242-4e11-9e48-e93df0d74333.png">

Reply:
I think field probably makes more sense here, but the semantics between field and property are a little vague to me outside of c# land

Reply:
I had the code from https://github.com/prisma/language-tools/issues/1347 for this.